src/js/push.js

```diff
@@ -203,6 +203,18 @@ class PushNotification {
 
     exec(successCallback, errorCallback, 'PushNotification', 'clearAllNotifications', []);
   }
+
+  /**
+   * Clears notifications that have the ID specified.
+   * @param  {[type]} id    ID of the notification to be removed.
+   */
+  clearNotification(id, successCallback = () => {}, errorCallback = () => {}) {
+    if (id && !isNaN(parseInt(id, 10))) {
```

Comment:
Considering the backend needs the `notId` to be an `int`, and even needs it to not go beyond `MAX_VALUE`, shouldn't we yield an error if the `id` is prone to error?

Reply:
Please look at my updated code, I have added more error checking (can't be below 0 or above MAX_SAFE_INTEGER) and added an error message. I don't believe there is any more error checking required on the back end, since the try catch will catch an error while parsing the integer out. Please let me know if these changes are sufficient.